pl_examples/full_examples/imagenet/dali_example.py

```diff
@@ -0,0 +1,36 @@
+import os
```

Comment:
Pls add docstring what is the purpose of DALI and refer resources
the point is that this example will be rendered along with other examples in the documentation